app/assets/javascripts/discourse/app/lib/keyboard-shortcuts.js

```diff
@@ -158,11 +160,10 @@ export default {
 
     this.keyTrapper?.destroy();
     this.keyTrapper = null;
-    this.container = null;
   },
 
   isTornDown() {
-    return this.keyTrapper == null || this.container == null;
+    return this.keyTrapper == null;
```

Comment:
`this.keyTrapper` is always nulled out together with `this.container` in `teardown` so this check is redundant